### DISCUSSION TITLE
HyperLink to open on a new tab

### DISCUSSION THREAD

**Discussion by joaoalbernaz** [QuestPDF/QuestPDF#515] [Ideas]
I think that a minor improvement which would help increase even more the quality of QuestPDF, is to allow the users to make the HyperLink open in a new tab, while in browsers. Since many users don't use a nativa PDF interpretator. 
Basically it would be a target "_blank", for the HyperLinks.

**Comment by girlpunk** [comment]
Does the PDF standard allow for that level of customisation? Remember that tabs are a browser invention, and not every PDF will be opened in a browser.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. As @girlpunk pointed out, the PDF standard does not support controlling how hyperlinks are opened — this behavior is determined by the PDF viewer, not the document itself. As such, this is outside the scope of QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
